app/assets/javascripts/discourse/app/widgets/post.js

```diff
@@ -15,10 +16,11 @@ import { transformBasicPost } from "discourse/lib/transform-post";
 import DiscourseURL from "discourse/lib/url";
 import { clipboardCopy, formatUsername } from "discourse/lib/utilities";
 import DecoratorHelper from "discourse/widgets/decorator-helper";
-import hbs from "discourse/widgets/hbs-compiler";
+import hbsCompile from "discourse/widgets/hbs-compiler";
```

Comment:
Elsewhere we've been using `widgetHbs` for this

```suggestion
import widgetHbs from "discourse/widgets/hbs-compiler";
```